<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QWidget" name="Logowanie">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Login:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditLogin"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Hasło:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditHaslo">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonZaloguj">
         <property name="text">
          <string>Zaloguj</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="PoZalogowaniu">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>181</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonPoZalDodaj">
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPoZalUsun">
         <property name="text">
          <string>Usun</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPoZalRaport1">
         <property name="text">
          <string>Raport1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPoZalRaport2">
         <property name="text">
          <string>Raport2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonWroc">
         <property name="text">
          <string>Wróć</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="DodajProdukt">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>691</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_25">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Nazwa</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodNazwa"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Jednostka podstawowa</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodJedPod"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Jednostaka w sztuce</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodJedSztu"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Opis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodOpis"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Typ</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodTyp"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Producent</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodProdu"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Podtyp</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodPodtyp"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Material</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodMat"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Kolor</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodKolor"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Dlugosc nominalna</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodDlugNom"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Szerokosc nominalna</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodSzerNom"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Grubosc</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodGrub"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22">
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Struktura</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodStrukt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Gradacja</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodGradacja"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Srednica</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditDodSrednica"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDodDodaj">
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonWrocZDodaj">
         <property name="text">
          <string>Wróć</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="uSUN">
     <widget class="QPushButton" name="pushButtonUsun">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>310</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Usun</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>420</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Wroc</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>285</width>
        <height>55</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Podaj id produktu opisowego do usunieica</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUsun"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="rap1">
     <widget class="QPushButton" name="pushButtonRap1Wroc">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>450</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Wroc</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>671</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rap2">
     <widget class="QTableView" name="tableView_rap2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>671</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRap2Wroc">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>430</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Wroc</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
